receiver/podmanreceiver/podman_client.go

```diff
@@ -56,11 +59,21 @@ type containerStatsReport struct {
 	Error string
 	Stats []containerStats
 }
+type Type = string
+type Actor struct {
+	ID         string
+	Attributes map[string]string
+}
+type Event struct {
+	dockerEvents.Message
```

Comment:
so far we've redefined structs in this package to avoid podman dependency so probably should do the same for docker